source/CommandLine/ShellCompletion/BashCompletionInstaller.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Octopus.CommandLine.Commands;
+using Octopus.CommandLine.Extensions;
+using Octopus.CommandLine.Plumbing;
+
+namespace Octopus.CommandLine.ShellCompletion
+{
+    public class BashCompletionInstaller : ShellCompletionInstaller
+    {
+        public override SupportedShell SupportedShell => SupportedShell.Bash;
+        public override string ProfileLocation => $"{HomeLocation}/.bashrc";
+        public override string ProfileScript
+        {
+            get
+            {
+                var sanitisedAppName = Path.GetFileName(ExecutablePaths.First()).ToLower().Replace(".", "_").Replace(" ", "_");
```

Comment:
The way we calculate these scripts has got more complicated now, as the app name was previously hardcoded, but now it's tweakable.